crates/services/p2p/src/gossipsub/config.rs

```diff
@@ -239,23 +241,43 @@ pub(crate) fn build_gossipsub_behaviour(p2p_config: &Config) -> Gossipsub {
     }
 }
 
-fn initialize_gossipsub(gossipsub: &mut Gossipsub, p2p_config: &Config) {
-    let peer_score_params = initialize_peer_score_params();
+const EXPECTED_TXS_PER_SLOT: f64 = 20.0;
```

Comment:
It is almost impossible to predict how many transactions we expect. It fully depends on `max_tx_limit` and `max_gas_limit`.

Reply:
we can have some healthy estimate of amount of tx messages we expect to receive from a single peer within a single SLOT, it does not have to be perfect

Reply:
But how does it affect gossiping? How will it behave if we have more than `20` transactions? And what if less? Sorry, I don't know how it works inside, and I'm trying to get it=)

Reply:
For a given N amount of messages in SLOT period, gossipsub will analyze and consequently score peers and their performances. For example if we say we expect 20 messages, and the peer keeps delivering way less than that then it's reputation will decrease. Having said that, if we struggle to have an estimate for txs (unlike for some other messages) then we can also disable peer scoring for this `topic`